packages/passport-crypto/src/passportCrypto.ts

```diff
@@ -32,6 +32,13 @@ export class PCDCrypto {
     return utils.arrayBufferToHexString(arrayBuffer);
   }
 
+  // Combines generateSalt and argon2 function, returns both salt and generated key
+  public generateSaltAndArgon2(password: Utf8String, length = 32) {
+    const salt = this.generateSalt();
+    const key = this.argon2(password, salt, length);
+    return { key, salt };
+  }
```

Comment:
I think we should abstract away the fact that this is even using anything called `Argon2` - the function could be something like `generateSaltAndEncryptionKey` and it would just take `password` as a parameter - the fact that this also takes a `length` parameter introduces a source of mistakes.